src/main/java/com/griddynamics/gridu/phonebookmvc/entity/Contact.java

```diff
@@ -0,0 +1,24 @@
+package com.griddynamics.gridu.phonebookmvc.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "CONTACT_TABLE")
+public class Contact {
+
+    @Id
+    @GeneratedValue
+    private int id;
+    private String name;
+    private String phones;
```

Comment:
Just curious, why ``String`` not ``List<String>``?

Reply:
I was used ``String`` to make DB more simple (Just one table without "1 to many" relationship). But I changed it to ``List<String>``.